ci-operator/step-registry/aws/deprovision/route53/private-hosted-zone/aws-deprovision-route53-private-hosted-zone-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+REGION="${LEASED_RESOURCE}"
+HostedZoneId="${SHARED_DIR}/hosted_zone_id"
+
+echo "Deleting AWS route53 hosted zone"
+if [ ! -f "${HostedZoneId}" ]; then
+  echo "File ${HostedZoneId} does not exist."
+  exit 1
+fi
+
+cmd="aws --region $REGION route53 delete-hosted-zone --id $(cat ${HostedZoneId}) | jq -r '.ChangeInfo.Id' | awk -F '/' '{printf \$3}'"
+change_id=$(eval "${cmd}")
+echo "Deleted route53 hosted zone."
+
+aws --region $REGION route53 wait resource-record-sets-changed --id ${change_id} &
+wait "$!"
+echo "Waited for route53 hosted zone deletion."
```

Comment:
"Waited for" is less explicit about success/failure, and if we get here, we know the wait was successful.  What about:

```sh
echo "AWS route53 hosted zone ${HOSTED_ZONE} successfully deleted."
```

or `...record sets successfully changed.` or something?

Reply:
Done